packages/ui/src/machines/authenticator/index.ts

```diff
@@ -64,50 +69,92 @@ export function createAuthenticatorMachine() {
           ],
         },
         signIn: {
-          entry: 'spawnSignInActor',
-          exit: stopActor('signInActor'),
+          initial: 'spawnActor',
+          states: {
+            spawnActor: {
+              always: { actions: 'spawnSignInActor', target: 'runActor' },
+            },
+            runActor: {
+              entry: 'clearActorDoneData',
```

Comment:
`ActorDoneData`, defined below, are any data that are passed between actors when we transition from one to another (e.g. `signIn` -> `signUp`).